sentence_transformers/models/Pooling.py

```diff
@@ -22,6 +22,15 @@ class Pooling(nn.Module):
     :param pooling_mode_lasttoken: Perform last token pooling, see https://arxiv.org/abs/2202.08904 & https://arxiv.org/abs/2201.10005
     """
 
+    POOLING_MODES = (
+        "cls",
+        "lasttoken",
+        "max",
+        "mean",
+        "meansqrt",

```

Comment:
I think we should use `mean_sqrt_len_tokens`. The inconsistency with `_` in `mean_sqrt_len_tokens` versus no `_` in `lasttoken` and `weightedmean` is pretty bothersome, so normally `meansqrt` would be a lot better, but we should probably stick with what we already had for backwards compatibility:

https://github.com/UKPLab/sentence-transformers/blob/e0d93bb377eca9e0c94da436f3f35c336e9c109d/sentence_transformers/models/Pooling.py#L95-L96

Reply:
Good catch. I missed this.